roles/node/defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+# maintains directories and paths
+node_home_dir: /opt/{{ service_identifier }}
+
+node_bin_dir: "{{ node_home_dir }}/bin"
+node_config_dir: "{{ node_home_dir }}/config"
+node_network_dir: "{{ node_config_dir }}/{{ namada[type].chain_id }}"
+node_data_dir: "{{ node_home_dir }}/data"
+node_wasm_dir: "{{ node_data_dir }}/wasm"
+node_cometbft_dir: "{{ node_home_dir }}/cometbft"
+node_env_dir: "{{ node_home_dir }}/environment"
+node_log_dir: "{{ node_home_dir }}/log"
+
+node_directories:
+  - "{{ node_home_dir }}"
+  - "{{ node_bin_dir }}"
+  - "{{ node_config_dir }}"
+  - "{{ node_data_dir }}"
+  - "{{ node_cometbft_dir }}"
+  - "{{ node_env_dir }}"
+  - "{{ node_log_dir }}"
+
+node_log_file_path: "{{ node_log_dir }}/{{ service_identifier }}.log"
+
+node_config_file_path: "{{ node_network_dir }}/{{ node_config_file_name }}"
+
+node_environment_general_file_path: "{{ node_env_dir }}/{{ node_environment_general_file_name }}"
+
+node_priv_validator_key_path: "{{ node_cometbft_dir }}/config/{{ node_priv_validator_key_file_name }}"
```

Comment:
lets create another variable  node_cometbft_config_dir for {{ node_cometbft_dir }}/config/